app/_layouts/docs-v2.html

```diff
@@ -4,38 +4,42 @@
 id: documentation
 ---
 
-{% assign latest_page_url = page.url | replace: page.kong_version, page.kong_latest.release %}
-{% capture latest_page_exists %}{% page_exists {{latest_page_url}} %}{% endcapture %}
+{% assign latest_page_url = page.url | replace: page.kong_version,
+page.kong_latest.release %}
+{% capture latest_page_exists %}{% page_exists {{latest_page_url}} %}
+{% endcapture %}
 
 <div class="page v2 {{ page.class }}">
 
   <div class="container">
-    {% include docs-sidebar.html %}
+    {% include_cached docs-sidebar.html %}
```

Comment:
This file will need adjusting before we can cache it. Right now, the first version of the sidebar that loads up is the same one that will persist between versions. 

For example: if you open Enterprise 2.4.x docs, then switch to 2.2.x and try to click a menu item, it opens the 2.4.x version of that link.

This is happening because of page-context assign variables:
https://github.com/benbalter/jekyll-include-cache#one-potential-gotcha

Reply:
This was resolved by passing in `kong_version` (even if we don't use it) https://github.com/mheap/docs.konghq.com/commit/3083ed65250a40e954683a155a528edceafc1530#diff-b76293185985e73d1cdbcb33314dad3913b088a48e4fdfa91c6608f0bbe31756R15